src/components/Chip.tsx

```diff
@@ -1,9 +1,85 @@
+export enum CHIP_VARIANTS {
+  SOLID = "bg-blue-400 dark:bg-green-400 hover:bg-blue-500 dark:hover:bg-green-500",
+  TRANSPARENT = "bg-2cg-1 hover:bg-gradient-to-r from-blue-500 to-blue-500 text-blue-500 dark:bg-2cg-3 dark:hover:bg-gradient-to-r dark:from-green-500 dark:to-green-500 dark:text-green-500",
+}
+
 interface ChipProps {
   label: string;
   onDelete?: () => void;
   className?: string;
+  variant: CHIP_VARIANTS;
+  disabled?: boolean;
 }
 
-export default function Chip({ label, onDelete, className }: ChipProps) {
-  return <></>;
+export default function Chip({
+  label,
+  onDelete,
+  className,
+  variant,
+  disabled,
+}: ChipProps) {
+  //solid chip
+  if (variant == CHIP_VARIANTS.SOLID)
+    return (
+      <div
+        className={`flex items-center text-body-lg text-white px-3 py-0.5 rounded-full transition duration-200 ease-in-out select-none
+      ${disabled ? "bg-gray-400 opacity-95 pointer-events-none" : variant}
+      ${className}`}
+      >
+        <svg
```

Comment:
I exported the svgs into separate files as custom components because I needed to apply a fill color and for some reason this isn't really supported by the Image component. Let me know if there is a better solution